spec/install_bash.sh

```
@@ -13,7 +13,7 @@ echo "Installing bash $INSTALL_BASH_VERSION..."
 
 mkdir -p bash
 cd bash
-wget -qN https://ftpmirror.gnu.org/bash/bash-$INSTALL_BASH_VERSION.tar.gz
+wget -qN https://mirror.cyberbits.eu/gnu/bash/bash-$INSTALL_BASH_VERSION.tar.gz
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)